fdbserver/BlobManager.actor.cpp

```diff
@@ -5041,6 +5047,26 @@ ACTOR Future<int64_t> bgccCheckGranule(Reference<BlobManagerData> bmData, KeyRan
 	return bytesRead;
 }
 
+// Check if there is any pending split. It's a precheck for manifest backup
+ACTOR Future<bool> hasPendingSplit(Reference<BlobManagerData> self) {
```

Comment:
This seems problematic. It's fine for now as a POC but we will have to work around this, we can't block a backup on this I think.

Reply:
that's the data consistency issue we discussed last week. I want to generate a manifest file with less inconsistency issues during restore. it's the first workable prototype. will make it better next PR.